src/inference/src/threading/ie_executor_manager.cpp

```diff
@@ -90,9 +89,32 @@ void ExecutorManagerImpl::clear(const std::string& id) {
     }
 }
 
+namespace {
+
+class ExecutorManagerHolder {
+    std::mutex _mutex;
+    std::weak_ptr<ExecutorManager> _manager;
+
+    ExecutorManagerHolder(const ExecutorManagerHolder&) = delete;
```

Comment:
probably these lines are not needed, `_mutex` is not copy-constructible, so default copy-constructor and copy-assignment are already deleted

Reply:
fixed